site/src/layouts/ImageList.js

```diff
@@ -89,7 +103,7 @@ class ImageList extends Component {
     this.onKeyDown = this.onKeyDown.bind(this)
 
     this.state = {
-      isExpanded: false,
+      isExpanded: props.alwaysExpand ? true : false,
```

Comment:
Can't you directly use `props.alwaysExpand`? If you need to cast to a boolean use `Boolean()`.